src/kirby/components/segmented-control/segmented-control.component.tns.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, EventEmitter, Output, Input } from '@angular/core';
```

Comment:
Don't split web + {N} implementations...

Reply:
I actually get why it's split (looking at the different implementations of `onSegmentTap ` and `onSegmentClick`). So, it looks valid here? If valid... please register element in {N}, eg::

```
registerElement('element name goes here', () => require('./segmented-control.component').SegmentedControlComponent);
```